backend/schema/errors.go

```diff
@@ -11,7 +11,14 @@ type Error struct {
 	Msg       string   `json:"msg" protobuf:"1"`
 	Pos       Position `json:"pos" protobuf:"2"`
 	EndColumn int      `json:"endCol" protobuf:"3"`
-	Err       error    `protobuf:"-"` // Wrapped error, if any
+}
+
+func (e *Error) ToProto() *schemapb.Error {
```

Comment:
Is this a `Node`? Possibly not? If it is though, this should return a `proto.Message` to comply with that interface.

Reply:
It's not, I went back and forth but since it's not actually an element of the schema it felt a little conceptually weird to make it a node. i don't feel strongly though